CleverPush/CleverPush/Source/CleverPushInstance.m

```diff
@@ -1200,6 +1209,136 @@ - (void)syncSubscription {
     }];
 }
 
+#pragma mark - Api call and fetch out the subscription data and sync with Failure block
```

Comment:
You copied the whole syncSubscription method. Instead we want to overload it, please check again. Otherwise we have a lot of duplicated code.